wiki/Article_Style_Guide/en.md

```diff
@@ -296,14 +345,19 @@ In instances where you are talking directly to the reader (e.g. tutorial page),
 
 ## Emphasizing
 
-Know that the overuse of emphasis will reduce its effectiveness!
+Know that the overuse/abuse of emphasis will reduce its effectiveness!
+
+When trying to emphasize words, do **not** use capital letters.
+Instead, use bold.
```

Comment:
L351: Rephrase. Suggested ``Instead, bold the word you're trying to emphasize.``